src/server/schema/resolvers/queries/listLives.ts

```diff
@@ -0,0 +1,25 @@
+import { getDatabaseContext} from '../../../database';
+import { GraphQLQueryResolvers, GraphQLLife } from '../definitions';
+
+const query: GraphQLQueryResolvers['listLives'] = async (root) => {
+    const { collections } = await getDatabaseContext();
+    const lifeCursorArray = await collections.lives.find({}).toArray();
+
+    if (lifeCursorArray.length == 0) {
+        throw new Error(`No lives found`);
+    }
```

Comment:
No need to return an error if there's no match. We can simply return an empty array instead